py.test --pdb not working if re-exec is used

When using python-libfaketime auto re-exec feature for loading libfaketime.so, py.test is no longer able to stop capturing stdin and so pdb quit instantly:

```Python traceback
$ py.test test_pdb_dropin.py 
re-exec with libfaketime dependencies
===================================================================== test session starts =====================================================================
platform linux -- Python 3.5.2, pytest-3.5.1, py-1.5.3, pluggy-0.6.0
rootdir: /home/emmanuel/projects/pytest-libfaketime, inifile:
plugins: libfaketime-0.1.0
collected 1 item                                                                                                                                              

test_pdb_dropin.py 
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB set_trace (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
--Return--
> /home/emmanuel/projects/pytest-libfaketime/test_pdb_dropin.py(3)test_will_fail()->None
-> import pdb; pdb.set_trace()
(Pdb) 
F                                                                                                                                    [100%]

========================================================================== FAILURES ===========================================================================
_______________________________________________________________________ test_will_fail ________________________________________________________________________

    def test_will_fail():
>       import pdb; pdb.set_trace()

test_pdb_dropin.py:3: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/lib/python3.5/bdb.py:52: in trace_dispatch
    return self.dispatch_return(frame, arg)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <pdb.Pdb object at 0x7f5f3b47e7f0>, frame = <frame object at 0x7f5f3b208c48>, arg = None

    def dispatch_return(self, frame, arg):
        if self.stop_here(frame) or frame == self.returnframe:
            # Ignore return events in generator except when stepping.
            if self.stopframe and frame.f_code.co_flags & CO_GENERATOR:
                return self.trace_dispatch
            try:
                self.frame_returning = frame
                self.user_return(frame, arg)
            finally:
                self.frame_returning = None
>           if self.quitting: raise BdbQuit
E           bdb.BdbQuit

/usr/lib/python3.5/bdb.py:96: BdbQuit
================================================================== 1 failed in 0.05 seconds ===================================================================
```

If py.test is run with `--capture=no` option (disabling capture of stdout/stderr streams), pdb works fine.

--- a/pytest_libfaketime.py
+++ b/pytest_libfaketime.py
@@ -3,7 +3,11 @@
 import os
 import libfaketime
 
-def pytest_configure():
-    libfaketime.reexec_if_needed()
-    _, env_additions = libfaketime.get_reload_information()
-    os.environ.update(env_additions)
+
+def pytest_configure(config):
+    needs_reload, env_additions = libfaketime.get_reload_information()
+    if needs_reload:
+        capture_manager = config.pluginmanager.getplugin('capturemanager')
+        capture_manager.stop_global_capturing()
+        libfaketime.reexec_if_needed()
+        os.environ.update(env_additions)

